src/protocols/bdx/BdxMessages.h

```diff
@@ -309,6 +310,28 @@ struct DataBlock : public BdxMessage
 using Block    = DataBlock;
 using BlockEOF = DataBlock;
 
+/**
+ * A struct for representing BlockQueryWithSkip messages
```

Comment:
Could you rephrase this comment to not re-read code? It seems obvious that `struct BlockQueryWithSkip` is a "struct for BlockQueryWithSkip"

Reply:
Removed the unnecessary comments